docs/react-v9/contributing/component-implementation-guide.md

```diff
@@ -0,0 +1,264 @@
+  - If you have access to the Fluent UI Internal Teams channel, post a link to the spec in the Convergence channel so people know to look at the spec. At-mention the reviewers to ensure it's on their radar.
+- The spec does not have to be final with all issues or open questions resolved before merging the first draft.
+  - If the open questions are large enough to warrant their own discussion, consider creating separate issues to track those open questions.
+
+# Phase 2: Initial Implementation
+
+Once the first draft of the spec has been reviewed, you can get started on the implementation.
+
+## Implementation
+
+**Requirements**
+
+- Use React hooks
+- Use makeStyles
+- Respect Figma tokens (and use the Provider)
+- Respect API principles, shorthands and slots handling
+- No dependency on v0/v7/v8
+- Add tsdoc `/** */` comments to all public APIs in your component
+
+You can follow examples of other converged components for the structure of the implementation, such as:
+
+- [Button](https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-button/src/components/Button)
+- [Avatar](https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-avatar/src/components/Avatar)
+
+## Documentation
+
+### Storybook stories
+
+The primary public documentation for our controls is in the form of storybook stories demonstrating the various features and recommended usage of the control. The stories are in the /stories subfolder of the component package. In general, each story should cover only one feature of a control, and follow best practices for using the control. The `yarn create-component` command creates a single story for each component. If you're building a larger, more complex, compositional component, it may not make sense to publish a story for every component, so some may just be slightly opinionated flavors of existing v9 components. Be sure to be intentional about the story you're telling around the main component experience.
+
+You can get started by looking at existing storybook stories:
+
+- [Button examples](https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-button/stories/Button)
+- [Avatar examples](https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-avatar/src/stories/Avatar)
+
+### Additional documentation
+
+- Documentation comments in code for all public APIs
+- A `README.md` file in the component package
+- Initial migration guide from v8 and v0
+
+<a id="unstable-release"></a>
+
+## Preview Release
+
+Once the component is functional, it can be released as an unstable preview. This will allow both `designers` to review the built component and `partners` to try it out and give early feedback.
+
+Tests are not required to be completed before releasing the component as unstable, but it is recommended to at least add conformance tests and some basic unit tests.
```

Comment:
Long term: Test patterns linkout would be great, a lot of common rules that can trigger and the answer is not always obvious (i.e. disabling static classnames).